.github/workflows/deploy-firebase-dapp.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Builds the static site via webpack in a container image,
+# then copies that directory to localhost and pushes to Firebase, at:
+# https://app.testnet.penumbra.zone
+name: Deploy static site
+on:
+  # Support ad-hoc runs via dispatch, so we can deploy from
+  # unmerged feature branches if necessary.
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  # Uncomment to run on PRs; useful while debugging.
+  # pull_request:
+jobs:
+  build:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the source code
+        uses: actions/checkout@v2
+
+      - name: Build static site
+        run: ./build-static-site
```

Comment:
The `build-static-site` script existed in the dapp repo (https://github.com/penumbra-zone/penumbra_dapp/blob/632df2679e03490a27f396d118eba11dbf79e202/build-static-site), but doesn't exist here. What is the equivalent command to export the static site docroot? Is it `npm run build`, which I see in the `package.json`? I also tried running that command from within the `apps/webapp/` subdir, but it's not clear to me where the HTML docroot is exported, if anywhere.

Reply:
From [discussion in Discord](https://discord.com/channels/824484045370818580/1044457038886998137/1156975423859732562), the answer is: 

```
# from repo root:
pnpm build
# docroot available at `./apps/webapp/dist/`
```

Reply:
@conorsch Is the logic ready for us to close this PR?

Reply:
As of https://github.com/penumbra-zone/web/pull/48/commits/80b10f2dda035a3f6aa347e98d46ea7dd5539084, yes, I think we should merge. However, I see a conflict: the `pnpm-lock.yaml` needs to be updated. I'm not sure how to do that, so I've not rebased this PR to resolve. As soon as the conflict is resolved, it's OK to merge this PR, and I'll follow up with next steps on the Firebase config.